spec/models/automated_checks/claim_verifiers/census_subjects_taught_spec.rb

```diff
@@ -179,15 +179,15 @@ module ClaimVerifiers
                   subject(:body) { note.body }
 
                   it "returns 'Eligible' with the school workforce census subjects", if: policy == EarlyCareerPayments do
-                    expect(subject).to eq("[School Workforce Census] - Eligible:\n<pre>Subject 1: Mathematics / Mathematical Development (Early Years)\nSubject 2: Statistics\n</pre>\n")
```

Comment:
Looking at this test and AC n.6, shouldn't we keep the multiple eligible subjects in this and the following expectations? Basically test that we get multiple rows and subjects that are eligible for one TRN